kube_apiserver_metrics/manifest.json

```diff
@@ -1,27 +1,27 @@
 {
-  "display_name": "Kube_apiserver_metrics",
-  "maintainer": "help@datadoghq.com",
-  "manifest_version": "1.0.0",
-  "name": "kube_apiserver_metrics",
-  "metric_prefix": "kube_apiserver.",
-  "metric_to_check": "kube_apiserver.go_goroutines",
-  "creates_events": false,
-  "short_description": "Collect metrics from the Kubernetes APIserver",
-  "guid": "406b274b-c44d-4499-a329-efd053b3f538",
-  "support": "core",
-  "supported_os": [
-    "linux"
-  ],
-  "public_title": "Datadog-Kube_apiserver_metrics Integration",
-  "categories": [
-    "containers"
-  ],
-  "type": "check",
-  "integration_id": "kube-apiserver-metrics",
-  "assets": {
-    "monitors": {},
-    "dashboards": {},
-    "service_checks": "assets/service_checks.json"
-  },
-  "is_public": false
+    "display_name": "Kube_apiserver_metrics",
```

Comment:
I believe this should be adjusted as well. It will be the name of the tile in datadog.